MiauAPI/Models/QueryObjects/WishListObject.cs

```diff
@@ -0,0 +1,19 @@
+using MiauDatabase.Enums;
+
+namespace MiauAPI.Models.QueryObjects;
+
+/// <summary>
+/// Represents the object in an WishList query.
+/// </summary>
+/// <param name="Id">The id of the WishList.</param>
+/// <param name="UserId">The id of the user the WishList is related to.</param>
+/// <param name="ProductId">The id of the product the WishList is related to.</param>
+
+public sealed record WishListObject
+{
+    public int Id { get; init; }
+    public int UserId { get; init; }
+    public int ProductId { get; set; }
+ 
+
+}
```

Comment:
Você pode reescrever esse `record` com [sintaxe posicional](https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/builtin-types/record#positional-syntax-for-property-definition).